src/Keboola/DbExtractor/Test/ExtractorTest.php

```diff
@@ -74,21 +85,21 @@ protected function getEnv(string $driver, string $suffix, bool $required = false
     {
         $env = strtoupper($driver) . '_' . $suffix;
         if ($required) {
-            if (false === getenv($env)) {
-                throw new \Exception($env . " environment variable must be set.");
+            if (false === (bool) getenv($env)) {
+                throw new \Exception($env . ' environment variable must be set.');
```

Comment:
Kdyz uz na to sahame, tak neco jako 'The environment variable "%s" must be set.'

Reply:
ok upravím to, tyhle věci jsem neupravoval ani jsem to nezkoumal jestli by se mohlo upravovat

Reply:
no, mozna mas pravdu a udelej to nekdy jindy zvlast :)